templates/layouts/configs/_save-sign-out-button.html

```diff
@@ -1,45 +1,37 @@
-{% if current_user.is_authenticated %}
-  {% if account_service_log_out_url %}
+{% if current_user.is_authenticated and not hide_signout_button %}
+
     {% if save_on_signout %}
```

Comment:
When is this actually false?

Reply:
It's not set on:
- the introduction template 
- any error pages where they are signed in e.g. 404s, 
- static pages

Reply:
Ah ya, makes sense 👍